src/components/teamMember.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react"
+import "./teamMember.scss"
+export default function TeamMember({ member }) {
+  const { name, role, descriptions, imageSrc } = member
+  const [expanded, setExpanded] = useState(false)
+  const collapsedClasses =
```

Comment:
I think instead of setting two different variables I would set this up as an array and use the `Array.join()` method to convert it to a string.

```suggestion
  const classes = [
      "bio-collapsible__collapsible-content",
      expanded ? "bio-collapsible__collapsible-content--expanded" : "bio-collapsible__collapsible-content--collapsed"
    ]
```